test/py/requirements.txt

```diff
@@ -22,4 +22,3 @@
 # THE SOFTWARE.
 #####################################################################################
 
-numpy==1.21.6
```

Comment:
This empty file should be removed.

Reply:
The function `add_py_venv_fixture` uses this requirements file for onnx and non-onnx invocations in `COMMAND ${PYTHON_EXECUTABLE} -m pip install -r ${REQUIREMENTS_FILE})`. Also, for future use in the virtual environments, its better to have a requirements file where packages can be added as needed by tests.

Reply:
If you feel strongly about removing this file, I could add a check in CMakeLists.txt to see if the requirements files exists and only then run `COMMAND ${PYTHON_EXECUTABLE} -m pip install -r ${REQUIREMENTS_FILE})`
In future, if a package needs to be added, the developer can create the requirements file and add in the package.
It might just be easier to leave the empty file, but I do not have a very strong preference about it. So open to your suggestions on this. @umangyadav @pfultz2

Reply:
yeah lets not keep empty files lying around

Reply:
Sounds good. Done.